com.microsoft.mrtk.input/Interactors/Poke/PokeInteractor.cs

```diff
@@ -52,44 +51,6 @@ protected virtual bool TryGetPokePose(out Pose pose)
 
         #endregion IHandedInteractor
 
-        #region IMRTKInteractorVisuals
-
-        [Header("Interactor visuals settings")]
-
-        [SerializeField]
-        [Tooltip("The visuals representing the interaction point, such as a cursor, donut, or other marker.")]
-        private GameObject touchVisuals;
-
-        /// <summary>
-        /// The visuals representing the interaction point, such as a cursor, donut, or other marker.
-        /// </summary>
-        public GameObject TouchVisuals { get => touchVisuals; set => touchVisuals = value; }
```

Comment:
I think my last concern about this pattern generally is how we expect interactables to obtain a reference to the visuals (mainly via `IXRCustomReticleProvider`) performantly. Hopefully they won't need to do `GetComponentInChildren` or something!

Reply:
Using the `IXRCustomReticleProvider interface`, it actually get handled all on the unity side. **Unfortunately** that implementation actually does make a `GetComponent` call, but on the interactor itself... meaning we might need to do some asset configuration shuffling. Nevertheless, its something I think it's better left to XRI's interface for them to improve on in the future.

Reply:
I think we should try to avoid a `GetComponent` call for this scenario, as it could be _somewhat_ frequent (on every hover?) 🤔 unless there's another pattern we can think of, it might be best to have a visuals/reticle reference on the interactor, since that's what the interactable gets passed (and I know I just approved #11043 haha, but I've been thinking through it a bit more and am starting to be less sure 😅)

Not a blocker for this PR though, but let's sync next week!